src/actors/containerization/services/wordpress/wordpress_scanner/wordpress_scanner.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+import re
+
+
+def get_version(root_dir):
+    pattern = '\d(.\d){1,2}'
+    version = ''
+
+    with open(os.path.join(root_dir, 'wp-includes/version.php')) as f:
+        for line in f:
+            if line.startswith('$wp_version'):
+                version = re.search(pattern, line).group(0)
+                break
+
+    return version
+
+
+def get_db_host(root_dir):
+    pattern = '(?<=DB_HOST\',)\s*[\w.\'()]+(?=\))'
+    with open(os.path.join(root_dir, "wp-config.php")) as f:
+        config = f.read()
+
+    match = re.search(pattern, config)
+
+    if match is None:
+        raise Exception("Unable to find address of database host")
+
+    host = match.group(0)
+    db_host = re.search('(?<=\')[\w.]+(?=\')', host).group(0)
+
+    if 'getenv' in host:
+        db_host = os.getenv(db_host, "")
+
+    if not db_host:
+        raise Exception("Unable to find address of database host")
+    else:
+        if db_host == 'localhost':
+            db_host = '127.0.0.1'
+        return db_host
+
+
+def get_root_dir():
+    root_dir = ''
```

Comment:
May i suggest you to first check default wordpress root dir? Then execute this brute force.
Btw what about augeas? Did you check if you can get some information about wp?